src/interface/src/app/treatments/treatment-area-layer/treatment-area-layer.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, inject, Input, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {
+  FeatureComponent,
+  GeoJSONSourceComponent,
+  LayerComponent,
+} from '@maplibre/ngx-maplibre-gl';
+import { PlanService } from '@services';
+import { Geometry } from 'geojson';
+import { GeoJSONSource, Map as MapLibreMap } from 'maplibre-gl';
+
+@Component({
+  selector: 'app-treatment-area-layer',
+  standalone: true,
+  imports: [FeatureComponent, GeoJSONSourceComponent, LayerComponent],
+  templateUrl: './treatment-area-layer.component.html',
+  styleUrl: './treatment-area-layer.component.scss',
+})
+export class TreatmentAreaLayerComponent implements OnInit {
```

Comment:
Can you rename this to PlanningAreaLayer ? This layer is showing the planning area

Reply:
Updating.....